### ISSUE TITLE
can u share all the network benchmark?

### ISSUE THREAD

**Issue by novioleo** [frotms/image_classification_mxnet_gluon#1]
thk~

**Comment by frotms** [comment]
@novioleo 
You can find them in the following URLs:

http://mxnet.incubator.apache.org/api/python/gluon/model_zoo.html

https://gluon-cv.mxnet.io/model_zoo/classification.html

**Comment by novioleo** [comment]
@frotms sorry i didn't find any benchmark of flowers102 with mobilenet.
i noticed that you have achieved approximate 90% on flowers102 validation dataset.
nowadays,i reach 90% acc on  flowers102 validation dataset,i don't know the best score of mobilenet on flowers102.

**Comment by frotms** [comment]
@novioleo 
There is no benchmark for flowers-102 datasets in these networks. You may train your model with tricks to get accuracy higher.

**Comment by novioleo** [comment]
@frotms can you share some your train tricks with me?
i had tried some many tricks,but the accuracy can't improve any more..

**Comment by frotms** [comment]
more augmentations, mixup, label smooth, change loss function, optimizer, learning rate scheduler, etc. @novioleo

**Comment by novioleo** [comment]
@frotms thks for your reply,i'll try ~

**Comment by novioleo** [comment]
thks for your tips , i imporved the acc to 92%... @frotms
